torch/random.py

```diff
@@ -129,21 +133,20 @@ def fork_rng(devices=None, enabled=True, _caller="fork_rng", _devices_kw="device
         return
 
     if devices is None:
-        num_devices = torch.cuda.device_count()
+        num_devices = device_mod.device_count()
```

Comment:
is `device_count()` listed on the list of methods that custom modules authors need to implement?